### DISCUSSION TITLE
Is there something similar to the CreateFreshApiToken for Laravel Sanctum?

### DISCUSSION THREAD

**Discussion by phppirate** [laravel/framework#33763] [General]
With Laravel Passport you have the ability to consume your Laravel API from the Laravel Front End using the default app.js file if you include the [`\Laravel\Passport\Http\Middleware\CreateFreshApiToken::class`](https://laravel.com/docs/7.x/passport#consuming-your-api-with-javascript) in your `web` middleware.

Is there something similar to this or some other way to achieve the same result with Laravel Sanctum?

**Comment by driesvints** [comment]
@phppirate not at this time no.

**Reply by ankurk91** [reply]
It would be nice addition to Sanctum

**Comment by iraklisg** [comment]
I was looking for that option today :yum: Indeed it would be a nice addition to Sanctum

**Comment by iraklisg** [comment]
Hello all and happy new year

I was wondering if there is any plans to implement this feature in the near future
